tests/parsers/test_utils.py

```diff
@@ -45,6 +45,14 @@ def entries():
     ]
 
 
+@pytest.fixture
+def entries_url_format():
+    return [
+        ParserEntry(name="foo", type="Method", path="foo%20bar.html#foo"),
+        ParserEntry(name="qux", type="Class", path="foo%20bar.html"),
```

Comment:
I don't think there's a point to making this a fixture? I think it can be safely inlined.

Reply:
I can just add a new entry in the `entries` fixture but I wanted to have a separate test just for checking URL-encoded filenames. I see know it can be done without a new fixture. Give me a sec.